test/python/ibmq/test_registration.py

```diff
@@ -143,6 +143,20 @@ def test_delete_all_accounts(self):
             self.assertEqual(len(qiskit.IBMQ._accounts), 0)
             self.assertEqual(len(read_credentials_from_qiskitrc()), 0)
 
+    @requires_qe_access
+    def test_pass_bad_proxy(self, qe_token, qe_url):
+        """Test proxy pass through."""
+        prox = {'urls': {
+            'http': 'http://user:password@1.2.3.4:5678',
+            'https': 'https://user:password@1.2.3.4:5678'}}
```

Comment:
Can you replace the host with either an invalid one, or one that resolves quickly: perhaps `127.0.0.1`? This way hopefully we avoid the delay, while still getting the error.

Reply:
ok